### PR TITLE
Add images to the introduction

### PR DESCRIPTION
**Pull request by avahoffman**
Images added for "what is scientific data" and "what are good excuses not to share"

image

image

### CODE REVIEW

File: 01-intro.Rmd [hunk 1]
```diff
@@ -84,7 +84,7 @@ The DMS Policy does not apply to research and other activities that do not gener
 - Resource (Gs)
 - Research-Related Infrastructure Programs (e.g., S06)
 
-You can look up your NIH Activity Code [here](https://sharing.nih.gov/sites/default/files/List-of-Activity-Codes-Applicable-to-DMS-Policy.pdf) to see if you are required to submit a DMS Plan.
+You can look up your NIH Activity Code [here](https://sharing.nih.gov/sites/default/files/List-of-Activity-Codes-Applicable-to-DMS-Policy.pdf){target="_blank"} to see if you are required to submit a DMS Plan.
```
Comment (carriewright11) [inline 01-intro.Rmd:87]: Nice!

File: index.Rmd [hunk 2]
```diff
@@ -32,7 +32,7 @@ In this course we will describe what the new policy requires, places where you m
 
 ## Motivation
 
-Navigating [https://sharing.nih.gov/](https://sharing.nih.gov/) can be confusing. We want to help make the process of compliance with this new policy easier. 
+Navigating [`https://sharing.nih.gov/`](https://sharing.nih.gov/) can be confusing. We want to help make the process of compliance with this new policy easier.
```
Comment (carriewright11) [inline index.Rmd:35]: hmmm should we maybe reword this a tad just to make it seem like we are friendlier to the NIH :P - Like navigating the new policy?

File: Appendix.Rmd [hunk 3]
```diff
@@ -3,5 +3,5 @@
 
 # FAQ
 
-The official NIH DMS Policy FAQ page can be found here: [https://osp.od.nih.gov/scientific-sharing/nih-policy-for-data-management-and-sharing-faq/](https://osp.od.nih.gov/scientific-sharing/nih-policy-for-data-management-and-sharing-faq/){target="_blank"}.
+The official NIH DMS Policy FAQ page can be found here: [`https://osp.od.nih.gov/scientific-sharing/nih-policy-for-data-management-and-sharing-faq/`](https://osp.od.nih.gov/scientific-sharing/nih-policy-for-data-management-and-sharing-faq/){target="_blank"}.
```
Comment (carriewright11) [inline Appendix.Rmd:6]: great to add this!

**Review by carriewright11 — APPROVED** [review]

File: index.Rmd (hunk 2 above)
Comment (avahoffman) [reply index.Rmd:35]: yes totally! good call

File: 01-intro.Rmd [hunk 4]
```diff
@@ -100,6 +100,9 @@ You are not expected to share:
 - communications with colleagues
 - physical objects (such as biospecimens)
 
+```{r, echo=FALSE, fig.alt='Scientific data should be shared under the new DMS policy, but there are many scientific products that are not considered scientific data.', out.width = '70%', fig.align = 'center'}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1luFoDzF6aDJEebbL6iWoJ_s8s9nQnaWLL5jghbmWdak/edit#slide=id.g1740f401082_0_0")
```
[the remark this answers is not in the document]
Comment (avahoffman) [reply 01-intro.Rmd]: unpublished results is in the image, but we can also emphasize elsewhere too!

File: 01-intro.Rmd [hunk 5]
```diff
@@ -100,7 +101,17 @@ You are not expected to share:
 - communications with colleagues
 - physical objects (such as biospecimens)
 
+<div class = "warning">
+
+You might be wondering if it is required to share raw data. 
+
+The DMS Policy applies to **final research data**. This is data that is recorded factual material commonly accepted in the scientific community as necessary to **validate and replicate** research findings. For many scientific areas, final research data includes both raw data and analyses conducted on the data. 
+
+For example, final research data for a clinical study would include a dataset that was used in the published study, but not the clinical documents (for example, medical records) that the dataset was derived from.
```
[the remark this answers is not in the document]
Comment (avahoffman) [reply 01-intro.Rmd:138]: I can comment it out for now until we know for sure :)
